src/languages/solidity.js

```diff
@@ -132,9 +132,10 @@ function hljsDefineSolidity(hljs) {
         excludeEnd: true,
         keywords: {
             built_in: 'gas value selector address length push pop ' +
-               'send transfer call callcode delegatecall staticcall ' +
-               'balance code codehash ' +
-               'name creationCode runtimeCode interfaceId min max'
+                'send transfer call callcode delegatecall staticcall ' +
+                'balance code codehash ' +
+                'wrap unwrap ' +
```

Comment:
This was pre-existing but it would be nice to have comments that explain where these groupings come from: Address / user types, type information etc

Reply:
Sure, I could add that!